packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx

```diff
@@ -593,8 +598,11 @@ export class VerticalStackedBarChartBase extends React.Component<
     color: string,
     refSelected: React.MouseEvent<SVGElement> | SVGGElement,
   ): void {
-    if (this._calloutAnchorPoint !== point) {
-      this._calloutAnchorPoint = point;
+    if (this._calloutAnchorPoint?.chartDataPoint !== point || this._calloutAnchorPoint?.xAxisDataPoint !== xAxisPoint) {
```

Comment:
What is the issue here. Can this be fixed just in the example

Reply:
The issue fixed here is for if the bar y value is same, the single callout does not move from the first instance where the callout was shown. This can't be fixed in the example

Reply:
Ok got it.